src/devices_models/devices/hydro_generation.jl

```diff
@@ -366,10 +382,14 @@ function add_constraints!(
     X::Type{<:PM.AbstractPowerModel},
 ) where {V <: PSY.HydroPumpedStorage, W <: AbstractHydroReservoirFormulation}
     if get_attribute(model, "reservation")
+        # TODO: Are the "reservation" models compatible with SC FF?
```

Comment:
@sourabhdalvi same here

Reply:
same response here, SC FF shouldn't be used here

Reply:
We should throw an exception here then, same as above.